src/app/helptext/task-calendar/replication/replication.ts

```diff
@@ -2,234 +2,243 @@ import { T } from '../../../translate-marker';
 
 export default {
     name_placeholder: T('Name'),
-    name_tooltip: T('Replication task name.'),
+    name_tooltip: T('Descriptive name for the replication.'),
 
     direction_placeholder: T('Direction'),
-    direction_tooltip: T('Direction of travel. <i>Push</i> sends\
- snapshots to a destination system. <i>Pull</i> receives snapshots from\
- a destination system.'),
+    direction_tooltip: T('Direction of travel. <i>Push</i> sends \
+ snapshots from the local system to either another dataset on the local \
```

Comment:
I feel like this overcomplicates this sentence and makes it more difficult to understand for the standard use case. This is a little clearer:
```
<i>Push</i> sends snapshots from the local system to a remote system, or to another dataset on the local system.
```
Best to fix the UI form so it does not show Direction when local->local is chosen. Please enter a ticket for that.

Reply:
https://jira.ixsystems.com/browse/NAS-103227